.pylintrc

```diff
@@ -0,0 +1,661 @@
+[MAIN]
+
+# Analyse import fallback blocks. This can be used to support both Python 2 and
+# 3 compatible code, which means that the block might have code that exists
+# only in one or another interpreter, leading to false positives when analysed.
+analyse-fallback-blocks=no
```

Comment:
nit: I would personally prefer to reduce the .pylintrc file to the settings that we are actually setting, which differ from the defaults - for me it's hard to tell which of the settings below were left unchanged from the defaults vs. which were adjusted for our repo 🤔